### ISSUE TITLE
Define imports are not being included

### ISSUE THREAD

**Issue by mihar-22** [vidstack/player#773]
B4154 on Discord:

> So using the library on Nuxt3 with the /define/* imports does not work due to tree shaking. `"sideEffects": false` in `package.json` causes this.

**Comment by BlakeB415** [comment]
Tree shaking doesn’t seem to work unless the module is explicitly referenced and used in the JS. Because of this, the define imports won’t be used.

To temporarily fix this, I removed the sideEffects field to prevent tree shaking on the package.

**Comment by thehydrogen** [comment]
Until a fix is pushed, you can patch Vidstack like so:

Create the following file: **patches/vidstack+0.2.3.patch**
```patch
diff --git a/node_modules/vidstack/package.json b/node_modules/vidstack/package.json
index e3da0bf..f4217f1 100644
--- a/node_modules/vidstack/package.json
+++ b/node_modules/vidstack/package.json
@@ -5,7 +5,7 @@
   "license": "MIT",
   "type": "module",
   "types": "dist/types/index.d.ts",
-  "sideEffects": false,
+  "sideEffects": true,
   "engines": {
     "node": ">=16"
   },
```

add `patch-package` to your dev dependencies

and add the following `postinstall` script to your `package.json`:
```
patch-package --exclude 'nothing'
```

**Comment by mihar-22** [comment]
This will be resolved in the next release `0.3.0` that's dropping this week 👍
